Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldUI.cs

```diff
@@ -180,6 +185,16 @@ public void Disable()
 		SwitchFocus (_CommanderUI);
 	}
 
+	public void _ArmouryUI_OnCurrencyChanged(int val, Player player)
+	{
+		_ArmouryUI.CurrencyChangedUpdate(val, player);
+	}
+
+	public void _ArmouryUI_OnShowToggled(bool toggledOn, Player player)
```

Comment:
You're naming these functions like event listeners? This sounds like you're listening for "_ArmouryUI_OnShowToggled" but you're actually calling the function. Again, you can get the active player from the game state so you just need the function to be "ShowAmoury()" and then you can have another for HideArmoury(). looks a lot cleaner ;)